OracleLinuxDevelopers/oraclelinux8/ocne-tools/1.8/help.sh

```diff
@@ -0,0 +1,9 @@
+#!/bin/bash
+
```

Comment:
Can you add a copyright header to this script please? Or perhaps just write this info out during build instead of running the script on every instantiation of a container.

Reply:
Done